app/src/androidTest/java/xyz/kotlout/kotlout/UserControllerFirebaseTest.java

```diff
@@ -0,0 +1,138 @@
+package xyz.kotlout.kotlout;
```

Comment:
Shouldn't this be in app/src/test/java/xyz/kotlout/kotlout?

Reply:
No, it requires context, so it needs to be here unless we use something like Robolectric.

Reply:
Ah makes sense.